pkg/ipamd/datastore/data_store.go

```diff
@@ -266,9 +274,19 @@ func (addr AddressInfo) Assigned() bool {
 	return !addr.IPAMKey.IsZero()
 }
 
+// getCooldownPeriod returns the time duration in seconds configured by the IP_COOLING_PERIOD env variable
+func (ds *DataStore) getCooldownPeriod() time.Duration {
+	cooldownVal, err := utils.GetIntAsStringEnvVar(envIPCooldownPeriod, 30)
+	if err != nil {
+		ds.log.Debugf("Error parsing cooldown period, using default cooldown period: 30")
+		return 30 * time.Second
+	}
+	return time.Duration(cooldownVal) * time.Second
```

Comment:
should be no need for `time.Duration` here. It is just an integer times `time.Second`

Reply:
Removing time.Duration results in this error `invalid operation: cooldownVal * time.Second (mismatched types int and time.Duration)`